motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -285,7 +299,7 @@ typedef struct _SmBodyMotoReadIOBit SmBodyMotoReadIOBit;
 struct _SmBodyMotoReadIOBitReply
 {
 	UINT32 value;
-	UINT32 resultCode;
+	IoResultCodes resultCode;

```

Comment:
Is this always correct? I'm not completely up-to-speed on how VxWorks maps enums to integer types, but afaik the standard does not necessarily dictate that 4-byte integers must be used for all enums.

Reply:
I'm not sure if that is dictated by the build system or the OS. But, I can confirm with certainty that an enum is a 4 byte int in this environment.

Reply:
Hm. I don't personally like to assume things like this (as they tend to change and break things while I'm not looking), but if you're comfortable with this, then we can keep it.